react/hooks/useMeasure.js

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react'
+import ResizeObserver from 'resize-observer-polyfill'
+
+export default function useMeasure(ref) {
```

Comment:
basically copy and pasted from here https://codesandbox.io/s/q3ypxr5yp4?from-embed